app/controllers/chat_rooms_controller.rb

```diff
@@ -19,6 +19,8 @@ def create
     end
 
     def show
+        @user = User.all
```

Comment:
It's helpful to name your variables with the same plurality as what they contain. This makes it easier elsewhere in the code to tell how to treat the data. Looking at @user in the view makes you think it's one model, not an array.